### PR TITLE
docs: serverless cli calls out including dagster-cloud

### PR DESCRIPTION
**Pull request by yuhan**
### Summary & Motivation
not sure if thats a good idea but it's easy to miss the line in the status quo 
https://dagster-git-yuhan-08-08-docsserverlessclicallso-a35e74-elementl.vercel.app/dagster-cloud/deployment/serverless#without-github-gitlab-bitbucket-or-local-development

### CODE REVIEW

**Comment by yuhan** [comment]
Current dependencies on/for this PR:
* master
  * **PR #9306** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/9306" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/9306?utm_source=stack-comment).

**Review by erinkcochran87 — APPROVED** [review]
Approving to unblock, JIC. Please check a live preview before merging - there may be a duplicate callout at the top of this page

**Pull request merged**
